script/catmux_create_session

```diff
@@ -103,8 +88,19 @@ def main():
 
     command = ['tmux']
     if args.tmux_config:
-        print('Using tmux config file: {}'.format(resolve_path(args.tmux_config)))
-        command += ['-f', resolve_path(args.tmux_config)]
+        tmux_config = args.tmux_config
+    elif os.path.exists(os.path.expanduser("~/.tmux.conf")):
+        tmux_config = os.path.expanduser("~/.tmux.conf")
+    elif os.path.exists("/etc/tmux.conf"):
+        tmux_config = "/etc/tmux.conf"
+    elif os.path.exists(os.path.join(site.USER_BASE, 'catmux', 'tmux_default.conf')):
+        tmux_config = os.path.join(site.USER_BASE, 'catmux', 'tmux_default.conf')
```

Comment:
```suggestion
    elif os.path.exists(os.path.join(site.USER_BASE, 'share', 'catmux', 'tmux_default.conf')):
        tmux_config = os.path.join(site.USER_BASE, 'share', 'catmux', 'tmux_default.conf')
```

This should fix the defaulting to the default tmux config if no config is provided or present for the `--user` case